From dcc3c0ea82c5b339138a0feb54baf97d3cbe8ce9 Mon Sep 17 00:00:00 2001
From: Tim Staley <tim.staley@soton.ac.uk>
Date: Mon, 14 May 2012 19:42:04 +0100
Subject: Ensure proper dependency check for BBSControl

BBSControl will not build without 'libpqxx3' - if it is present, the source list is altered.
Previously, if libpqxx3 was missing, the build would fail
with a cryptic error message at link time.
---
 CEP/Calibration/BBSControl/CMakeLists.txt |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CEP/Calibration/BBSControl/CMakeLists.txt b/CEP/Calibration/BBSControl/CMakeLists.txt
index 60b50ed..0004c30 100644
--- a/CEP/Calibration/BBSControl/CMakeLists.txt
+++ b/CEP/Calibration/BBSControl/CMakeLists.txt
@@ -6,7 +6,7 @@ lofar_package(BBSControl 1.0 DEPENDS BBSKernel Blob Common LMWCommon ParmDB PLC
 include(LofarFindPackage)
 lofar_find_package(Boost REQUIRED)
 lofar_find_package(Casacore REQUIRED COMPONENTS casa measures)
-lofar_find_package(PQXX)
+lofar_find_package(PQXX REQUIRED)
 
 add_subdirectory(include/BBSControl)
 add_subdirectory(src)
-- 
1.7.9.5

